source/include/core_sntp_client.h

```diff
@@ -192,7 +196,12 @@ typedef int32_t ( * UdpTransportSendTo_t )( NetworkContext_t * pNetworkContext,
  * @note It is RECOMMENDED that the read operation is non-blocking, i.e. it
  * SHOULD immediately return when no data is available on the network.
  * In such a case, the coreSNTP library will re-try send operation for a maximum period
- * of time configured in the SNTP_RECV_POLLING_TIMEOUT_MS macro.
+ * of blocking time passed through the @ref Sntp_ReceiveTimeResponse API.
+ *
+ * @note If the size of the SNTP response packet from the server exceeds the
+ * Maximum Transmission Unit (MTU) supported by the network interface of the device,
+ * the user-defined implementation of the interface SHOULD support receiving and assembling
+ * fragmented UDP packets.
```

Comment:
I'm not sure if SHOULD is the right wording here because the Linux documentation suggests decreasing the packet size instead of relying on fragmentation.

Reply:
Have updated the sentence to suggest customer to handle that scenario either by supporting fragmentation operations OR using packet size that fits within an MTU sized frame